standalone/rewrite_config.go

```diff
@@ -32,28 +33,48 @@ func main() {
 
 	//separate the lines in each node log
 	for i := 0; i < (len(node_logs) - 1); i++ {
+		var swarm_port string
 		var api_port string
 		var remote_api_port string
 
 		//separate the lines on newline character
 		lines := strings.Split(node_logs[i], "\n")
 		for j := 0; j < (len(lines)); j++ {
+			if strings.HasPrefix(lines[j], "Swarm listening on /ip4/") {
+				//split to find port value
+				parts := strings.Split(lines[j], "/")
+				if len(parts) > 0 {
+					swarm_port = parts[(len(parts))-1]
```

Comment:
Suggest add below code to guarantee the port is a number 
v, err := strconv.Atoi(s)
		if err != nil {
			log.Panic("The port cannot be read", zap.Error(err))
		}

Reply:
@JustinGaoF  can you take a look, i added the check for the port number.